docs/application/tizen-studio/rt-ide/overview.md

```diff
@@ -1,7 +1,7 @@
 # TizenRT
 
 > [!NOTE]
-> Tizen RT IDE has been deprecated since Tizen Studio 4.1 release, but you can still browse the obsolete RT-IDE documentation using the [link](/application/tizen-studio/rt-ide/get-started/overview.md).  
+> Tizen RT IDE has been deprecated since Tizen Studio 4.1. You can access the obsolete RT IDE documentation [here](/application/tizen-studio/rt-ide/get-started/overview.md). 
```

Comment:
I think so much would do

(get-started/overview.md)

Reply:
@annie-samsung This is working as is on staging, tried changing but the link broke. 
https://docs1.stg.tizen.org/staging/1356/application/tizen-studio/rt-ide/overview/ --> https://docs1.stg.tizen.org/application/tizen-studio/rt-ide/get-started/overview

Reply:
@nawab-samsung when you change and check on staging, you still have to apply PR number and staging in URL

Reply:
As @annie-samsung replied, each staging link for PRs has the PR number in the URL.

docs1.stg.tizen.org/**staging**/**1356**/application/tizen-studio/rt-ide/overview/

In the hyperlink if you start the URL with `/application...` as @nawab-samsung did,
the hyperlink goes to the "docs.tizen.org + /application...".

In case of https://docs1.stg.tizen.org/staging/1356/application/tizen-studio/rt-ide/overview/,
as the starting point is https://docs1.stg.tizen.org,
if you make a hyperlink like \[here](**/application/tizen-studio/rt-ide/get-started/overview**.md),
the link goes to https://docs1.stg.tizen.org + **/application/tizen-studio/rt-ide/get-started/overview**,
not to the https://docs1.stg.tizen.org/staging/1356/application/tizen-studio/rt-ide/get-started/overview/.


These URLs are not same, 
so @nawab-samsung, if you check the https://docs1.stg.tizen.org/application/tizen-studio/rt-ide/get-started/overview,
you'd not see the RT IDE deprecated note that you added at rt-ide/get-started/overview.md.
To check your note that you added, https://docs1.stg.tizen.org/staging/1356/application/tizen-studio/rt-ide/get-started/overview/ is needed.

If this PR is merged, at that time, every URLs will work on docs.tizen.org. So it will work fine at the end, but till then checking the links can cause confusion.

So, please use Relative path when you add a link. 
In this case also, \[here](./get-started/overview.md) is recommended.

Reply:
Thanks, both of you.